lambdas/process-journey-event/src/main/resources/statemachine/journey-maps/new-p2-identity.yaml

```diff
@@ -358,31 +343,18 @@ states:
     nestedJourney: KBVS
     exitEvents:
       fail-with-no-ci:
-        targetState: PYI_CRI_ESCAPE
-        checkIfDisabled:
-          f2f:
-            targetState: PYI_CRI_ESCAPE_NO_F2F
+        targetState: PYI_KBV_DROPOUT_PHOTO_ID
       end:
-        targetState: PYI_CRI_ESCAPE
-        checkIfDisabled:
-          f2f:
-            targetState: PYI_CRI_ESCAPE_NO_F2F
+        targetState: PYI_KBV_DROPOUT_PHOTO_ID
       next:
         targetJourney: EVALUATE_SCORES
         targetState: START
       enhanced-verification:
-        targetState: MITIGATION_02_OPTIONS_WITH_F2F
+        targetState: PYI_KBV_DROPOUT_PHOTO_ID
```

Comment:
Hmm don't we still need a separate state for this, as the page will be context dependent - like `PYI_KBV_DROPOUT_NO_PHOTO_ID` and `MITIGATION_KBV_FAIL_NO_PHOTO_ID` for no photo id?